src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/register_app_interface_request.h

```diff
@@ -89,6 +89,9 @@ class RegisterAppInterfaceRequest
    **/
   void SendRegisterAppInterfaceResponseToMobile();
 
+  void FillApplicationParams(
```

Comment:
@LuxoftAKutsan please provide some description for this function
BTW why it is in public section?